### ISSUE TITLE
possible position cheat in your example 08-centered-auto.vue ?

### ISSUE THREAD

**Issue by TimothyFehr** [surmon-china/vue-awesome-swiper#80]
Hello,
I am experiencing some position issues when auto positioning my elements and figured out, that there is a strong consonance between the number in the "spaceBetween" field and the number of pixels my whole slideshow moves after the first switch.

I see in your code that you apparently sized the three elements differently (but they do not appear in different sizes in your slideshow). So i guess you Bugfixed the issue in a specific way to resolve your demo. Sadly I can not do that, since the number of elements in my case is dynamic. 

My educated guess would be that you simply forgot to calculate the spaceBetween in your switch to the next element functionality.

Is it possible to resolve that issue?

Thanks in advance

**Comment by chengmac** [comment]
what?

**Comment by TimothyFehr** [comment]
Maybe a image is helpful to you.

<img width="1258" alt="bildschirmfoto 2017-05-04 um 08 39 54" src="https://cloud.githubusercontent.com/assets/9962501/25692801/79ed3310-30a5-11e7-8a5c-741108a1cd46.png">

I instantiate 2 times the same slider, it works well but once I move the slider (as I did with the second one) the center point of the slider is moved. And as far as I can see its exactly the amount which I speechified in the "spaceBetween" field. I already tested it and if a set it to 0 it does not move the center point.

I hope this clarifies my encountered issue.
